packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -407,6 +407,55 @@ Clients should be allowed to use arbitrary content for the display part of a hea
 </nimble-table>
 ```
 
+### Validation
```

Comment:
It may make sense to move the validation spec updates to the validation PR since that is already open and likely to diverge somewhat from here. Eitherway by the end we should have this up to date 👍

Reply:
I'm just going to update it here as part of this PR.